### PR TITLE
Upload coverage reports to codecov, fix macOS tests

### PR DESCRIPTION
**Pull request by samuel-w**
It hasn't been running for a month https://app.codecov.io/gh/borgbackup/borg/commits?page=1

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -110,7 +110,7 @@ jobs:
         brew install zstd || brew upgrade zstd
         brew install lz4 || brew upgrade lz4
         brew install openssl@1.1 || brew upgrade openssl@1.1
-        brew install Caskroom/cask/osxfuse || brew upgrade Caskroom/cask/osxfuse  # Required for Python llfuse module
+        brew install homebrew/cask/osxfuse || brew upgrade homebrew/cask/osxfuse  # Required for Python llfuse module
```
Comment (ThomasWaldmann) [inline .github/workflows/ci.yml:114]: so caskroom is old style (not working any more?) and homebrew/cask is new style?

**Review by ThomasWaldmann — COMMENTED** [review]
can you fix the commit comments?

i compile the changelog from these and just "update ci" doesn't say much.

**Comment by samuel-w** [comment]
The title can be used as the commit comment

File: .github/workflows/ci.yml (hunk 1 above)
Comment (samuel-w) [reply .github/workflows/ci.yml:114]: https://github.com/borgbackup/borg/pull/5537/checks?check_run_id=1491240535
Otherwise it breaks

Comment (ThomasWaldmann) [reply .github/workflows/ci.yml:114]: ok

**Comment by ThomasWaldmann** [comment]
thanks!

**Pull request merged**